mapiproxy/servers/default/asyncemsmdb/dcesrv_asyncemsmdb.c

```diff
@@ -806,10 +817,10 @@ static int process_newmail_notification(TALLOC_CTX *mem_ctx,
 static void EcDoAsyncWaitEx_handler(struct tevent_context *ev,
 				    struct tevent_fd *fde,
 				    uint16_t flags,
-				    void *private_data)
+				    void *asyncemsmdb_session)
 {
-	struct asyncemsmdb_private_data			*p = talloc_get_type(private_data,
-									     struct asyncemsmdb_private_data);
+	struct exchange_asyncemsmdb_session		*p = talloc_get_type(asyncemsmdb_session,
+									     struct exchange_asyncemsmdb_session);
 	TALLOC_CTX					*mem_ctx;
```

Comment:
Can you revert these variable renaming changes. These are imho cosmetic changes and it makes it more difficult for me to review the actual session change being discussed and its scope.